sql/catalyst/src/test/scala/org/apache/spark/sql/UnsafeProjectionBenchmark.scala

```diff
@@ -0,0 +1,107 @@
+package org.apache.spark.sql
+
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.catalyst.encoders.RowEncoder
+import org.apache.spark.sql.catalyst.expressions.UnsafeProjection
+import org.apache.spark.sql.types._
+import org.apache.spark.util.Benchmark
+
+object UnsafeProjectionBenchmark {
```

Comment:
yea, I think it is fine (as long as we provide enough doc on how to run it). We also have `ParquetReadBenchmark` in the repo.